ansible-playbook.yml

```diff
@@ -6,11 +6,11 @@
     - name: secret_key
       prompt: What is the app secret key?
     
-    - name: trello_api_key
-      prompt: What is the Trello API key?
+    - name: db_connection_string
+      prompt: What is the primary database connection string?
 
-    - name: trello_token
-      prompt: What is the Trello token?
+    - name: db_name
+      prompt: What is the database name?
```

Comment:
Nice work updating these - for the record we won't  be reusing Ansible so no harm in letting them drift, but it's even better if we can keep them in line! 😄